bibtex-actions-file.el

```diff
@@ -124,21 +123,24 @@ If you use 'org-roam' and 'org-roam-bibtex', you can use
             (when file-field
               (seq-mapcat
                (lambda (func)
-                 (funcall func dirs file-field))
+                 (funcall
+                  func
+                  ;; Make sure this arg is non-nil.
+                  (or dirs default-directory)
+                  file-field))
                bibtex-actions-file-parser-functions))))
       (append results-key results-file))))
 
-(defun bibtex-actions-file--files-for-entry (entry dirs extensions)
-    "Find files related to ENTRY in DIRS with extension in EXTENSIONS."
+(defun bibtex-actions-file--files-for-entry (key entry dirs extensions)
+    "Find files related to KEY, ENTRY in DIRS with extension in EXTENSIONS."
     (seq-filter #'file-exists-p
-                (bibtex-actions-file--possible-names entry dirs extensions)))
+                (bibtex-actions-file--possible-names key entry dirs extensions)))
 
-(defun bibtex-actions-file--files-to-open-or-create (keys dirs extensions)
-  "Find files related to a list of KEYS in DIRS with extension in EXTENSIONS."
-  (cl-flet ((files-for-key
-             (key)
+(defun bibtex-actions-file--files-to-open-or-create (key entry dirs extensions)
+  "Find files related to a KEY, ENTRY in DIRS with extension in EXTENSIONS."
+  (cl-flet ((files-for-key ()
```

Comment:
This strange now, since the function definition in `cl-flet` takes no argument and the final result is just running it. The `cl-flet` here can be removed and the function body will then be just the sexp with `let*`.